04 - Guides, Workflows, & Courses/Guides/How to debug why Obsidian is running slowly.md

```diff
@@ -46,13 +46,13 @@ In the event that you're experiencing slowness _even with plugins disabled,_ the
 
 A performance snapshot will tell you exactly what operations/functions are taking up all the CPU cycles when Obsidian is being slow.
 
-To take a performance snapshot, first open the Developer Tools (`View › Toggle Developer Tools`). Find the "Performance" tab.
+To take a performance snapshot, first open the Developer Tools (<kbd>Ctrl</kbd>-<kbd>Shift</kbd>-<kbd>I</kbd> on Windows/Linux or <kbd>Cmd</kbd>-<kbd>Opt</kbd>-<kbd>I</kbd> on MacOS; on MacOS alternatively `View › Toggle Developer Tools`). Find the "Performance" tab.
```

Comment:
@liamcain On Windows and Linux there is no `View > Toggle Developer Tools`, so I added the shortcuts.

Reply:
Ah, fair enough. Good catch!